File: Atributos de validación Data Annotations multiidioma/[C#]-Atributos de validación Data Annotations multiidioma/C#/LocalizeDataAnnotations/Controllers/HomeController.cs
using LocalizeDataAnnotations.Infrastructure;
using LocalizeDataAnnotations.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace LocalizeDataAnnotations.Controllers
{
    public class HomeController : Controller
    {

        public ViewResult Index(Persona persona)
        {
            return View(persona);
        }

        public ViewResult Resource(PersonaResource persona)
        {
            return View("Index", persona);
        }

        public ViewResult BBDD(PersonaBBDD persona)
        {
            return View("Index", persona);
        }

        public ViewResult CustomAttr(PersonaCustomAttr persona)
        {
            return View("Index", persona);
        }

    }
}

File: Atributos de validación Data Annotations multiidioma/[VB]-Atributos de validación Data Annotations multiidioma/VB/LocalizeDataAnnotations/Controllers/HomeController.vb
Imports System.Web.Mvc

Namespace Controllers
    Public Class HomeController
        Inherits Controller

        Function Index(persona As Persona) As ViewResult
            Return View(persona)
        End Function

        Function Resource(persona As PersonaResource) As ViewResult
            Return View("Index", persona)
        End Function

        Function BBDD(persona As PersonaBBDD) As ViewResult
            Return View("Index", persona)
        End Function

        Function CustomAttr(persona As PersonaCustomAttr) As ViewResult
            Return View("Index", persona)
        End Function

    End Class
End Namespace